lerobot/common/robot_devices/utils.py

```diff
@@ -1,19 +1,37 @@
-import platform
 import time
 
 
-def busy_wait(seconds):
-    if platform.system() == "Darwin":
-        # On Mac, `time.sleep` is not accurate and we need to use this while loop trick,
-        # but it consumes CPU cycles.
-        # TODO(rcadene): find an alternative: from python 11, time.sleep is precise
-        end_time = time.perf_counter() + seconds
-        while time.perf_counter() < end_time:
+def precise_sleep(seconds: float, blocking: bool = False):
```

Comment:
I'm wondering if this should default to `blocking = True`, and have the fancy non-cpu-blocking version for advanced users.